mmedit/edit.py

```diff
@@ -0,0 +1,248 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import os
+import warnings
+import torch
+from typing import Dict, List, Optional, Union
+
+from mmedit.apis.inferencers import MMEditInferencer
+from mmedit.apis.inferencers.base_mmedit_inferencer import InputsType
+from mmedit.utils import register_all_modules
+
+
+class MMEdit:
+    """MMEdit API for mmediting models inference.
+
+    Args:
+        model_name (str): Name of the editing model. Default to 'FCE_IC15'.
```

Comment:
A confused point may be where should users find the ``model_name`` for their need. 
Maybe we can offer a detailed list or description.

Reply:
right, we should add an interface to print all models that are available for inference